lib/src/base-styles/_nx-colors.scss

```diff
@@ -55,9 +55,11 @@
   --nx-color-alert-warning: var(--nx-swatch-orange-50);
   --nx-color-alert-error: var(--nx-swatch-red-40);
 
-  //NxBinaryDonutChart variables
-  --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-95);
```

Comment:
You can't get rid of these.  All CSS variables are part of our public API.  They should be marked as deprecated in the docs (and in code comments here) and their values should be updated to be aliases for the equivalent new variables.

Reply:
> You can't get rid of these. All CSS variables are part of our public API. They should be marked as deprecated in the docs (and in code comments here) and their values should be updated to be aliases for the equivalent new variables. The deprecation notice in the docs should indicate what variables should be used instead of these.

I reinstated the CSS variables with a code comment saying they're deprecated here: [187794c](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/187794cd83ee1fde3a8c4cc310d56fa788250463)
I updated the binary donut doc to mention the deprecated variables here: [61f945d](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/61f945dab4ba32087fd0d54e9aba2662acf69ce0)

Reply:
> I updated the binary donut doc to mention the deprecated variables here: [61f945d](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/61f945dab4ba32087fd0d54e9aba2662acf69ce0)

That's not where I meant.  You should undo that, and add the appropriate documentation to the Style Variables page

Reply:
> > I updated the binary donut doc to mention the deprecated variables here: [61f945d](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/61f945dab4ba32087fd0d54e9aba2662acf69ce0)
> 
> That's not where I meant. You should undo that, and add the appropriate documentation to the Style Variables page

I reverted it here: [21c7ab3](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/21c7ab3aab045e0cc80a025059b2a2ebfbf29c83)

Reply:
You should be able to delete and not deprecate, as mentioned below: https://github.com/sonatype/sonatype-react-shared-components/pull/1049#discussion_r1161870886

Reply:
> You should be able to delete and not deprecate, as mentioned below: [#1049 (comment)](https://github.com/sonatype/sonatype-react-shared-components/pull/1049#discussion_r1161870886)

I removed the NxBinaryDonutChart variables here:
[94450a4](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/94450a4ad22d9420aaa210de2850301d9afae49d)
[bca6dfe](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/bca6dfee2d2c6757ecaf6af7179fe665d58dee8e)